There is a failure on the CI:

```
runtime/shared_heap.c:82:5: error: incompatible integer to pointer conversion initializing '_Atomic(pool *)' with an expression of type 'int' [-Werror,-Wint-conversion]
  { 0, },
    ^
runtime/shared_heap.c:83:5: error: incompatible integer to pointer conversion initializing '_Atomic(pool *)' with an expression of type 'int' [-Werror,-Wint-conversion]
  { 0, },
```

--- a/Changes
+++ b/Changes
@@ -614,6 +614,9 @@ Working version
 - #12727, #12730: fix bug with value let-rec and labelled applications
   (Vincent Laviron, review by Gabriel Scherer)
 
+- #12755: Fix data race on global pools arrays of pool_freelist
+  (Fabrice Buoro and Olivier Nicole, review by Gabriel Scherer)
+
 OCaml 5.1.1
 -----------
 

--- a/runtime/shared_heap.c
+++ b/runtime/shared_heap.c
@@ -79,8 +79,8 @@ static struct {
   CAML_PLAT_MUTEX_INITIALIZER,
   NULL,
   { 0, },
-  { 0, },
-  { 0, },
+  { NULL, },
+  { NULL, },
   NULL
 };
 

